### PR TITLE
Add functionality to buttons

### PR DESCRIPTION
**Pull request by David-Lanzz**
# HELLO THERE👋👋👋
extra-perfect-added

## I have completed the tasks assigned to me for today, below is the summary of what I have done👍

- I have added functionality to all buttons according to their functions.👍💯
- I have added a module to the website.👍💯
- No linter errors
- Correct use of GitHub flow
## Any suggestions, insights, or recommendations? Please leave a comment on my issues page, thankyou 😊😊

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import './style.css';
+import { addlists, List } from './modules/functionality.js';
+
+const outputarray = JSON.parse(localStorage.getItem('inputarray'));
+
+window.addEventListener('click', (e) => {
+  if (e.target.id === 'edit') {
+    e.target.parentNode.firstElementChild.nextElementSibling.toggleAttribute('readonly');
+    e.target.parentNode.firstElementChild.nextElementSibling.classList.toggle('edited');
```
Comment (levy002) [inline src/index.js:9]: - [ ] Kindly make sure once the task description is edited, you save the new description in the local storage and make sure that change will persist until we change it again. It means even if we can reload the page we will get the new edit edited description.
   
    ### YOUR APP BEHAVIOUR
   chrome-capture-2023-1-23

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  } else if (e.target.id === 'addbtn') {
+    addlists();
```
Comment (levy002) [inline src/index.js:11]: - [OPTIONAL] Kindly make sure to clear the `input` after adding a new task in order to achieve a better user experience.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @David-Lanzz  , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights
- GOOD UI :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to create the `Remove` task functionality which will allow the user to be able to remove any task he wants to and make sure after deleting the task the local storage is updated as well.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @David-Lanzz    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**